docker-compose.yaml

```diff
@@ -1,6 +1,6 @@
+---
 services:
   app:
     image: ghcr.io/kuwairo/angular-react-starter:latest
     restart: on-failure
-    ports:
-    - 127.0.0.1:8087:8080
+    ports: [127.0.0.1:8087:8080]
```

Comment:
Why it fixed by yamlfix in such a way? Can we change it to:

```
ports:
  - "8087:8080"
```

Reply:
Done